examples/advanced/bionemo/downstream/tap/jobs/fedavg_tap_esm1nv/app1/custom/downstream_flip_tap.yaml

```diff
@@ -1,34 +1,56 @@
 name: esm1nv_flip
 defaults:
   - pretrain_small
+  - _self_
+do_preprocessing: False
 do_training: True # set to false if data preprocessing steps must be completed
 do_testing: False # set to true to run evaluation on test data after training
-restore_from_path: /bionemo_nvflare_examples/bionemo/models/esm1nv.nemo # path to nemo checkpoint of the protein model
+restore_from_path: null # path to nemo checkpoint of the fine-tuned model (encoder + task head) to be used for further training, testing or inference
 target: bionemo.model.protein.esm1nv.ESM1nvModel # target class for protein model
 infer_target: bionemo.model.protein.esm1nv.infer.ESM1nvInference # target inference class for protein model
+encoder_frozen: False
+
+data:
+  task_name: tap # options: aav, bind, conservation, gb1, meltome, sav, scl, secondary_structure
+  task_type: regression #'token-level-classification'  # alternative: classification, regression
+  preprocessed_data_path: /tmp/data # path where all preprocessed FLIP datasets are saved
+  dataset_path: ${model.data.preprocessed_data_path}/tap # path to a training data
+  dataset:
+    train: tap_full_train
+    val: tap_valid
+    test: tap_test
+  sequence_column: "Antibody" # name of column with protein sequence in csv file
+  target_column: ["PPC"] #["3state", "resolved"] # names of label columns in csv file
+  target_sizes: [1] # number of classes in each label for classifications or 1 for regression
+  num_workers: 8
+  shuffle: True # shuffle training dataset
+  max_seq_length: ${model.seq_length}
+  emb_batch_size: ${model.micro_batch_size}
+  batch_size: 128
 
 trainer:
   devices: 1 # number of GPUs or CPUs
   num_nodes: 1 
-  max_epochs: 1
+  max_epochs: 200
```

Comment:
for FedAvg, the local training should happen for `max_epochs: 1` and several FL rounds. The effective training epochs will be local epochs (i.e. max_epochs) * FL rounds (specified in config_fed_server.conf)

Reply:
cool changed that